noir_stdlib/src/array.nr

```diff
@@ -5,23 +5,53 @@ impl<T, N> [T; N] {
     #[builtin(array_len)]
     pub fn len(self) -> Field {}
 
-    #[builtin(arraysort)]
-    pub fn sort(self) -> Self {}
+    pub fn sort(self) -> Self {
+        self.sort_via(|a, b| a <= b)
+    }
```

Comment:
We should add a `where` clause on this to ensure the generic T has an ordering function. Not sure why this isn't caught already.
```suggestion
    pub fn sort(self) -> Self where T: Ord {
        self.sort_via(|a, b| a <= b)
    }
```

Reply:
Done